src/screens/Library.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { useState, useEffect } from 'react';
+
+import Card from "../components/Card";
+import Loader from "../components/Loader";
+import Button from "../components/Button";
+
+import getDataFromApi from '../utils';
+
+
+const Library = (props) => {
+    // Toggle State
+    const [toggle, updateToggle] = useState(false);
+
+    // API URL by Toggle State
+    const apiUrl = toggle ?
```

Comment:
Usually you want Components to be as "dumb" as possible. That means they should not be aware of API endpoints and other stuff like that.

I would move this whole statement in the `utils.js`. One thing you can also do is separate the API host, since it'll always be the same (`"https://match-labs-api.herokuapp.com/api/no_auth"`) for each endpoint. One advantage of doing it this way is that if you need to modify the host, you'll only have to do it once!